css/save-animation.css

```diff
@@ -0,0 +1,34 @@
+.eu-cookie-compliance-save-preferences-button {
```

Comment:
Let's give the css file name something that resembles the module - `eu-cookie-compliance.css` or something similar. I can imagine we will have CSS in here that is not just about the animation.